int/api/webcreator.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func PrepareForWebsiteHandler(params operations.WebsiteCreatorPrepareParams) middleware.Responder {
-	address, err := website.PrepareForUpload(*params.Body.URL)
+	address, err := website.PrepareForUpload(*params.Body.URL, *params.Body.Nickname)
```

Comment:
if you add to those mandatory fields the entry x-nullable: false in the swagger configuration file, go-swagger will make them as true object instead of pointer.
Maybe go-swagger goes even further by automatically handling error (this need to be check :-).